robot_controllers/src/parallel_gripper.cpp

```diff
@@ -57,10 +57,51 @@ int ParallelGripperController::init(ros::NodeHandle& nh, ControllerManager* mana
 
   // Setup Joints */
   std::string l_name, r_name;
-  nh.param<std::string>("l_gripper_joint", l_name, "l_gripper_finger_joint");
-  nh.param<std::string>("r_gripper_joint", r_name, "r_gripper_finger_joint");
-  left_ = manager_->getJointHandle(l_name);
-  right_ = manager_->getJointHandle(r_name);
+
+//Checking to see if the joints have been set in the parameter server
+//Warns and attempts to use default joint names
+
+  if(!nh.hasParam("l_gripper_joint"))
+    {  
+     ROS_ERROR_NAMED("ParallelGripperController",
```

Comment:
I think this is probably a warning instead of an error level.  It probably a good practice to set this value, but not really horrible if it is not set.